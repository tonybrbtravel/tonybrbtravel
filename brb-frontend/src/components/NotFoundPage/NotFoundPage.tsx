import { Card, Grid } from 'semantic-ui-react';

export const NotFoundPage = () => (
  <Grid centered columns={3}>
    <Grid.Column>
      <Card>Sorry, could not find that page.</Card>
    </Grid.Column>
  </Grid>
);
