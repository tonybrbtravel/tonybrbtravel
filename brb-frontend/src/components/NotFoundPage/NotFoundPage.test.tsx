import { render } from '@testing-library/react';
import NotFoundPage from '.';

describe('<NotFoundPage />', () => {
  it('should match snapshot', () => {
    const { container } = render(<NotFoundPage />);
    expect(container).toMatchSnapshot();
  });
});
