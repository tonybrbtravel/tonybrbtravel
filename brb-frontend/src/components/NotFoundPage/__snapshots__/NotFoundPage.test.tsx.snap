// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage /> should match snapshot 1`] = `
<div>
  <div>
    <div
      class="ui container module-container"
    >
      <div
        class="p-os-header-menu"
      >
        <div
          class="title-section"
        >
          <div
            class="ui mini breadcrumb"
          >
            <div
              class="divider"
            >
              /
            </div>
            <a
              class="section"
            >
              Growth
            </a>
          </div>
          <h1
            class="ui center aligned header"
          >
            Not found
          </h1>
        </div>
        <div
          class="p-os-header-menu__bottom"
        >
          <div
            class="ctaContent"
          >
            <div />
          </div>
        </div>
      </div>
      <main
        class=""
      >
        <div
          class="ui centered three column grid"
        >
          <div
            class="column"
          >
            <div
              class="ui card"
            >
              Sorry, could not find that page.
            </div>
          </div>
        </div>
      </main>
    </div>
  </div>
</div>
`;
